test/gtest/uct/test_sockaddr.cc

```diff
@@ -380,14 +384,14 @@ class test_uct_sockaddr : public uct_test {
         }
     }
 
-    virtual void server_accept(entity *server, uct_conn_request_h conn_request,
+    virtual void server_accept(uct_conn_request_h conn_request,
                                uct_cm_ep_server_conn_notify_callback_t notify_cb,
                                uct_ep_disconnect_cb_t disconnect_cb,
                                void *user_data, bool can_fail)
     {
         ucs::scoped_async_lock listen_lock(m_server->async());
-        ucs::scoped_async_lock accept_lock(server->async());
-        accept(server->cm(), conn_request, notify_cb, disconnect_cb,
+        ucs::scoped_async_lock accept_lock(m_server->async());
```

Comment:
something wrong here, AFAIR double locking was added because listen and accept servers were different for ep migration test

Reply:
I'll not investigate the problem of double locking in this PR

Reply:
changing of virtual method semantic should go thru all the hierarchy, otherwise this is not what was initially expected

Reply:
I changed `server_accept` for all the hierarchy. It is initially defined in `test_uct_sockaddr`.

Reply:
I see change in 1 from 3 implementations

Reply:
I missed 2 others, so need to refactor in separate PR